### DISCUSSION TITLE
Setting Header of Object to Collapsed State

### DISCUSSION THREAD

**Discussion by qscheitlin** [UI5/webcomponents-react#4704] [Q&A] [answered]
Hi All, 

This might seem like a silly question, but after looking through the documentation and the q&a forum, I couldn't find an answer.  Is it possible to preset the object page header to be in a collapsed state.  I see there is a 'onToggleHeaderContent' event, but that event is only fired when the header is content is manually pinned. 

Thank you,
Quintin

**Comment by Lukas742** [comment, marked as answer]
Hi @qscheitlin 

currently it is not possible to toggle the ObjectPage header programmatically. Since there is already an open [feature request](https://github.com/SAP/ui5-webcomponents-react/issues/4206) asking for this behavior, I will close this question thread. If you have additional questions or information, please feel free to raise them in the linked feature request.

Duplicate #4206
